core/src/main/java/com/datastrato/gravitino/lock/TreeLock.java

```diff
@@ -84,8 +87,21 @@ public void lock(LockType lockType) {
     for (int i = 0; i < length; i++) {
       TreeLockNode treeLockNode = lockNodes.get(i);
       LockType type = i == length - 1 ? lockType : LockType.READ;
-      treeLockNode.lock(type);
-      heldLocks.push(treeLockNode);
+      try {
+        treeLockNode.lock(type);
+        heldLocks.push(Pair.of(treeLockNode, type));
+        LOG.trace("Locked node: {}, lock type: {}", treeLockNode, type);
```

Comment:
Shall we also add this to like `if (LOG.isTraceEnabled())`?

Reply:
The log message's logic "if(Log.isTrace())") will be helpful if it has complex compute/IO logic or string concat, or if it doesn't have much affection.

Reply:
Yes, I also consider that this log does not have much overhead, such as string concat, so didn't add judgment statement
I can add this to keep consistent.